source/macos/AdaptiveCards/AdaptiveCards/AdaptiveCard.swift

```diff
@@ -38,11 +38,60 @@ open class AdaptiveCard {
 }
 
 public struct RenderConfig {
-    public static let `default` = RenderConfig(isDarkMode: false)
-    
+    public static let `default` = RenderConfig(isDarkMode: false, buttonConfig: .default)
     let isDarkMode: Bool
+    let buttonConfig: ButtonConfig
     
-    public init(isDarkMode: Bool) {
+    public init(isDarkMode: Bool, buttonConfig: ButtonConfig) {
         self.isDarkMode = isDarkMode
+        self.buttonConfig = buttonConfig
+    }
+}
+
+public struct ButtonColorConfig {
+    public static let `default` = ButtonColorConfig()
+    // backgroundColor
+    let backgroundColor: NSColor
+    
+    // buttonColor
+    let buttonColor: NSColor
```

Comment:
```suggestion
    // buttonColor
    let buttonColor: NSColor
```
`buttonColor` and `backgroundColor` seems to do the same job?

Reply:
yes, but when the mouse is exited, the buttonColor takes up the background color, will remove it from visualizer, there it is not required